src/draco/psy/psy_draco_decoder.cpp

```diff
@@ -228,14 +249,18 @@ class MeshDecompression::Impl
     }
 
     Header mDecompressedHeader;
+	uint32_t mCurrIFrameIndex;
     std::shared_ptr<::draco::Mesh> mpMesh;
     std::shared_ptr<::draco::DecoderBuffer> mpBuffer;
     std::unique_ptr<MeshEdgeBreakerDecompression> mpMeshDecompression;
     ::draco::Status mStatus;
 };
 
 MeshDecompression::MeshDecompression(const MeshDecompression&) : mpImpl(nullptr) {};
-MeshDecompression& MeshDecompression::operator=(const MeshDecompression&) { return *this; };
+MeshDecompression& MeshDecompression::operator=(const MeshDecompression&)
+{
+    return *this;
+};
```

Comment:
Do you know why we have this weird copy constructor?

Reply:
you mean assignment operator, and I do not @nbcong any input why you added it?